pygmt/tests/test_datasets_load_remote_datasets.py

```diff
@@ -32,10 +32,8 @@ def test_load_remote_dataset_benchmark_with_region():
     assert data.attrs["horizontal_datum"] == "WGS84"
     assert data.gmt.registration == 0
     assert data.shape == (11, 21)
-    # The cpt attribute was added since GMT 6.4.0
     # Can't access the cpt attribute using virtual files
-    # if Version(__gmt_version__) >= Version("6.4.0"):
-    #    assert data.attrs["cpt"] == "@earth_age.cpt"
+    # assert data.attrs["cpt"] == "@earth_age.cpt"
```

Comment:
Is this something that could be obtained from the `GMT_GRID` struct somehow?

Reply:
No, the cpt information is stored in the private, hidden structure `GMT_GRID_HEADER_HIDDEN`:

https://github.com/GenericMappingTools/gmt/blob/426abad288ef4b87c2cecf4c9633fd849866e45d/src/gmt_hidden.h#L163

We can't access it unless we decide to wrap the non-public GMT structure.